contributions/catalog/webapps/templates/contribute/home.html

```diff
@@ -307,41 +305,42 @@ <h4 class="talent_name"></h4>
     </div>
 
     <script>
+        function start(){
+
+            ajax('GET', get_cap_url, null,
+                    function (res) {
+                        var items = JSON.parse(res);
+                        all_capability = items
+                        append_capability(all_capability);
+                        filterSelection('all');
```

Comment:
Duplicate function calls in lines 315 and 316. One of these can be removed, right?

Reply:
Removed. (sorry I missed this one earlier)

Reply:
No issues. Thanks.